.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -0,0 +1,203 @@
+        - Something else
+    validations:
+      required: true
+  - type: input
+    id: docker-version
+    attributes:
+      label: Docker version
+      description: Please show us the output of `docker --version` on the host your mail server is running on if you are using Docker. Note that if you are not using Docker, there is no official support provided by maintainers as DMS mainly focuses on Docker (and Docker Compose).
+      placeholder: Docker version 20.10.7, build f0df350
+    validations:
+      required: false
+  - type: input
+    id: docker-compose-version
+    attributes:
+      label: Docker compose version
+      description: Please show us the output of `docker-compose --version` on the host your mail server is running on. Note that if you are not using Docker Compose, there is no official support provided by maintainers as DMS mainly focuses on Docker COmpose (and Docker).
+      placeholder: docker-compose version 1.28.6, build 5db8d86f
+    validations:
+      required: false
+  - type: input
+    id: uname-a
+    attributes:
+      label: The output of `uname -a`
+      description: Please paste the output of `uname -a` here.
+      placeholder: Linux swift 5.11.0-31-generic \#33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
+    validations:
+      required: true
+  - type: textarea
+    id: important-environment-variables
+    attributes:
+      label: Important environment variables
+      description: If there are environment variables important to this issue, please post them here. If we ask for the values of variables, make sure to put them in here too.
+      render: bash
+  - type: textarea
+    id: relevant-log-output
+    attributes:
+      label: Relevant log output
+      description: Show us relevant log output here. You can enable debug output by setting `DMS_DEBUG` to `1`.
+      render: shell
+  - type: textarea
+    id: other-relevant-information
+    attributes:
+      label: Other relevant information
+      description: If there is more, you can tell us here. Make sure to use Markdown formatting here.
+      render: Markdown
+  - type: dropdown
+    id: experience
+    attributes:
+      label: What level of experience do you have with Docker and mail servers?
+      description: You are not obliged to answer this question. But answering it helps us better understand where possible causes of errors lie. Less experienced users are more likely do make "obvious" mistakes, which is fine, and by answering this question, we know to look for them as well. We encourage everyone to participate, regardless of their level of experience.
+      options:
+        - I'm very much a beginner
+        - I'm a novice
+        - I know containerization but have less experience with mail
+        - I have experience with mail but less experience with containerization
+        - I know both pretty well and I'm experienced
+        - I'm an expert
+        - Trust me, I'm a (computer) engineer
```

Little rephrasing, if you're ok with being a little less formal. The expert option seemed a bit redundant so I folded it into the last one.

```suggestion
      options:
        - I have no idea what I'm doing! (beginner)
        - I know the basics, I've used Docker before (novice)
        - I'm comfortable with Docker / containers, but I'm relatively new to all this mail stuff
        - I have experience with mail servers, but less experience working with containers
        - I know both pretty well actually (experienced)
        - Trust me, I'm a (computer) engineer! (expert)
```

---

I like it :D